app/file-detail/template.hbs

```diff
@@ -1,29 +1,161 @@
-{{title model.name}}
-<div class='row'>
-    <div class='col-sm-5'>
-        <h2>{{model.name}}</h2>
+{{title model.file.name}}
+{{quickfile-nav user=model.user onQuickfiles=false}}
+<div class='quickfiles-content row'>
+    <div class='col-sm-8'>
+        <h2>{{model.file.name}} <a id='versionLink' onclick={{action 'changeView'}}>{{concat "(Version: " mfrVersion ")"}}</a></h2>
     </div>
-    <div class='col-sm-7'>
-        <div class='pull-right'>
-            <button {{action 'download'}} class='btn btn-sm btn-primary'>
-                Download
-            </button>
+    <div class='col-sm-4'>
+        <div id='toggleBar' class='pull-right'>
+            <div class='btn-toolbar m-t-md'>
+                <div class='btn-group m-l-xs m-t-xs'>
+                    <button class='btn btn-sm btn-primary popover-toggler' {{action 'share'}}>
```

Comment:
Clicking to open the popover triggers copying due to action share here. Is that intended? If it is, I would disagree that this behavior is desired, as it is not intuitive.